lib/OpenQA/Worker.pm

```diff
@@ -32,6 +32,11 @@ use OpenQA::Worker::Pool qw(lockit clean_pool);
 use OpenQA::Worker::Jobs;
 use OpenQA::Setup;
 
+# to be replaced manuallly when changing the server to keep compability.
+# If this value is lower than $OpenQA::WebSockets::Server::WORKERS_REQUIRED_VERSION
+# Then it won't be able to connect.
+our $VERSION = 1_0_0_0;
```

Comment:
I would make it a constant instead and group them into a single one,  indicating exactly that is a protocol version constant.

I say that mainly for two reasons : we don't want  to modify it runtime and if we have to replicate some constant,  we need to enforce the rules inside a test file (so we don't accidentally forget to bump it manually).
Besides in case the worker has a older version,  it will have this number lower.

Reply:
@mudler Thanks. I made them constant, however while trying to group both variables into one, i ran into some problems regarding the imports, that would make the tests to fail. So i kept two constants.